django/db/backends/sqlite3/base.py

```diff
@@ -27,13 +30,30 @@
 from .schema import DatabaseSchemaEditor                    # isort:skip
 
 
+if settings.DEBUG:
+    sqlite3.enable_callback_tracebacks(True)
+else:
+    sqlite3.enable_callback_tracebacks(False)
+
+
 def decoder(conv_func):
     """
     Convert bytestrings from Python's sqlite3 interface to a regular string.
     """
     return lambda s: conv_func(s.decode())
 
 
+def none_guard(func):
+    """
+    If any of the arguments to a function is None then it returns None.
+    """
+    def wrapper(*args, **kwargs):
+        if any(arg is None for arg in itertools.chain(args, kwargs.values())):
```

Comment:
I don't think that SQLite can pass kwargs, so I think that this can be replaced by `if None in args`.

Reply:
Ah. Yes, quite right. So this could be simplified to:
```python
return None if None in args else func(*args, **kwargs)
```